tests/chainer_tests/initializer_tests/test_constant.py

```diff
@@ -1,11 +1,12 @@
 import unittest
 
-from chainer import backend
+from chainer import backend, using_device
```

Comment:
We do not allow importing non-module objects, as per the OpenStack coding guideline.